definitions/checks/foreman_tasks/not_paused.rb

```diff
@@ -9,12 +9,20 @@ class NotPaused < ForemanMaintain::Check
     end
 
     def run
-      paused_tasks_count = feature(:foreman_tasks).paused_tasks_count(ignored_tasks)
+      paused_tasks_count = feature(:foreman_tasks).paused_tasks_count()
       assert(paused_tasks_count == 0,
              "There are currently #{paused_tasks_count} paused tasks in the system",
-             :next_steps =>
-               [Procedures::ForemanTasks::Resume.new,
-                Procedures::ForemanTasks::UiInvestigate.new('search_query' => scoped_search_query)])
+             :next_steps => next_procedures)
+    end
+
+    def next_procedures
+      if assumeyes?
+        [Procedures::ForemanTasks::Resume.new,
```

Comment:
Can you explicitly return the array inside if block ? now even when we have assumeyes? true it returns three next steps.